Exec/RegTests/Spray-Conv/SprayParticlesInitInsert.cpp

```diff
@@ -9,18 +9,13 @@ SprayParticleContainer::injectParticles( // NOLINT
   amrex::Real /*dt*/,
   int /*nstep*/,
   int /*lev*/,
-  int /*finest_level*/,
-  ProbParmHost const& /*prob_parm*/,
-  ProbParmDevice const& /*prob_parm_d*/)
+  int /*finest_level*/)
 {
   return false;
 }
 
 void
-SprayParticleContainer::InitSprayParticles(
-  const bool init_parts,
-  ProbParmHost const& /*prob_parm*/,
-  ProbParmDevice const& /*prob_parm_d*/)
```

Comment:
As discussed [here](https://github.com/AMReX-Combustion/PeleMP/pull/17/files#r1424431642), we should add a pp query in this function for "parcel_size", and add that as the last (optional) argument to unifromSprayInit, which would allow this test to recover its behavior its behavior from before we started all this reorganization.

Reply:
uh. Can you make that happen ;) ? I am lost

Reply:
done

Reply:
thank you!